app/assets/stylesheets/custom.css.scss

```diff
@@ -73,7 +77,7 @@ a:hover, a:active {
     padding-left: 10px;
 }
 
-#sign-out:hover {
+#sign-out:hover, #sign-out:focus {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line